etna/ciim/utils.py

```diff
@@ -205,7 +205,9 @@ def resolve_links(markup):
     def link_from_span(span):
         if link := pq(span).attr("link"):
             if iaid := re.match(r"\$link\((?P<iaid>[C0-9]*)\)", link).group("iaid"):
-                url = reverse("details-page-machine-readable", kwargs={"iaid": iaid})
+                url = reverse(
+                    "details-page-machine-readable", kwargs={"metadata_id": iaid}
+                )
```

Comment:
There's still a weird mix of terminology here, which I think could cause confusion

Reply:
Going with discussions.